test/utils/safeImageUrl.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Safe Image Url
+ *
+ * When using dummy image urls via faker in an img tag, if the
+ * component you're testing gets mounted karma will try to load that url.
+ * This makes a call out to the internet, which is unnecessary and seems to
+ * always crash phantomjs.
+ *
+ * Karma is configured to serve images locally from the /test/images folder,
+ * see the documentation for more details:
+ * https://github.com/karma-runner/karma/blob/master/docs/config/02-files.md#loading-assets
+ */
+
+/**
+ * Currently always returns the same one image but could be updated to
+ * randomize if we want to add more images to that folder.
+ *
+ * @returns {string} local url
+ */
+const safeImageUrl = () => '/base/test/images/logo.png'
```

Comment:
I think I'd prefer to simplify things for images instead.  Since we don't need the images to actually render, how about just using `foo.js` instead of real faker urls or the safeImageUrl?